### PR TITLE
feat: typescript conversion on video-intelligence

### PR DESCRIPTION
**Pull request by summer-ji-eng**
docs-test fails due to broken link, fixing in https://critique-ng.corp.google.com/cl/298705338

Other than that, it is ready to review.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import * as v1 from './v1';
+import * as v1beta2 from './v1beta2';
+import * as v1p1beta1 from './v1p1beta1';
+import * as v1p2beta1 from './v1p2beta1';
+import * as v1p3beta1 from './v1p3beta1';
+const VideoIntelligenceServiceClient = v1.VideoIntelligenceServiceClient;
+const StreamingVideoIntelligenceServiceClient =
+  v1p3beta1.StreamingVideoIntelligenceServiceClient;
+export {
```
[the remark this answers is not in the document]
Comment (summer-ji-eng) [reply src/index.ts]: StreamingVideoIntelligenceServiceClient is new Client introduced in v1p3beta1. I am not 100% sure if we want put it in export default. @alexander-fenster could you give me suggestion? Thanks

File: package.json [hunk 2]
```diff
@@ -30,31 +29,44 @@
...
+    "prelint": "cd samples; npm link ../; npm i",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5",
+    "google-gax": "^1.14.2",
     "protobufjs": "^6.8.8"
```
[the remark this answers is not in the document]
Comment (summer-ji-eng) [reply package.json]: thanks for pointing out. :)

**Comment by xiaozhenliu-gg5** [comment]
Seems these two tests fail for `samples-test`:
```
at Context.it (system-test/analyze_person_detection_gcs.test.js:29:12)
analyzing people in video
    1) should identify people in a local file
    2) should identify people in a file in Google Storage
```
And there's assertion error, that means, it got some results which is different from the expectation. Please debug a little bit, it might be useful to trace back when it starts to fail, and what changes did you make. Thank you!

File: synth.py [hunk 3]
```diff
@@ -5,30 +5,28 @@
...
+            "grpc-service-config": f"google/cloud/videointelligence/{version}/videointelligence_grpc_service_config.json",
+            "package-name": f"@google-cloud/video-intelligence",
+            "main-service": f"videointelligence",
+        },
+        proto_path=f'/google/cloud/videointelligence/{version}',
     )
 
     # skip index, protos, package.json, and README.md
-    s.copy(library, excludes=["package.json", "README.md", "src/index.js", "smoke-test/video_intelligence_service_smoke_test.js"])
+    s.copy(library, excludes=["package.json", "README.md", "src/index.ts",
```
Comment (JustinBeckwith) [inline synth.py]: Are we still generating smoke tests like this?  Seems snowflakey :)

Comment (alexander-fenster) [reply synth.py]: This is not a real "smoke test", it's a system test that is not named properly. @summer-ji-eng The new generator does not make any smoke tests, so we should just "upgrade" the generated smoke test to a system test and commit it into `system-tests` folder once and forever.

**Pull request merged**